assets/src/edit-story/app/media/media3p/actions.js

```diff
@@ -26,3 +26,10 @@ export const setSelectedProvider = (dispatch) => ({ provider }) => {
     payload: { provider },
   });
 };
+
+export const search = (dispatch) => ({ searchTerm }) => {
```

Comment:
Other action dispatchers are named as 'set*' etc when setting state, for example setMediaType, setProcessing and setSearchTerm in local.js.

Can this be named similarly, so setSearchTerm / types.SET_SEARCH_TERM?